modules/nw-sriov-dpdk-base-workload.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-sriov-dpdk-base-workload_{context}"]
+= Example DPDK base workload
+
+The following is an example of a Data Plane Development Kit (DPDK) container:
+[source,yaml]
+----
+    ---
```

Comment:
I just noticed that this whole example is indented. Should the top level start with no indentation and everything else align from that? Please check with an SME or QE reviewer.